_posts/2025-04-06-sm9276.md

```diff
@@ -8,7 +8,7 @@ layout: post
 
 # and specify a custom title
 
-# title: "Sample Blog Post"
+# title: "Profanity: XMPP Client Documentation"
```

Comment:
this should not start with a # (which is a comment) if you want it to be the title that gets used